nilearn/_utils/docs.py

```diff
@@ -329,37 +339,39 @@ def custom_function(vertices):
         - If a nonzero scalar is given, width is identical in all 3 directions.
         - If a :class:`numpy.ndarray`, :obj:`tuple`, or :obj:`list` is given,
           it must have 3 elements, giving the :term:`FWHM` along each axis.
-          If any of the elements is zero or None, smoothing is not performed
-          along that axis.
-        - If `fwhm='fast'`, a fast smoothing will be performed with a filter
+          If any of the elements is ``0`` or ``None``,
+          smoothing is not performed along that axis.
+        - If `fwhm="fast"`, a fast smoothing will be performed with a filter
           [0.2, 1, 0.2] in each direction and a normalisation to preserve the
           local average value.
-        - If ``fwhm`` is None, no filtering is performed (useful when just
-          removal of non-finite values is needed).
+        - If ``fwhm`` is ``None``, no filtering is performed
+          (useful when just removal of non-finite values is needed).
 
     .. note::
 
-        In corner case situations, `fwhm` is simply kept to None when `fwhm`
-        is specified as `fwhm=0`.
-
+        In corner case situations, `fwhm` is simply kept to ``None``
+        when `fwhm` is specified as `fwhm=0`.
```

Comment:
or actually maybe the single because then they are better differentiated from types which usually have this box around them. No strong preference either way

Reply:
On top of the visual distinctiveness, there are laziness and "linting" advantages:

single backtick --> less typing, shorter lines

Reply:
Ok let's go with that then!

Reply:
OK will try to apply it in this PR

Reply:
should have fixed most of them